docs/non_containerized_NSFS.md

```diff
@@ -612,7 +612,9 @@ Noobaa logs are pushed to `var/log/noobaa.log` and the log is rotated and compre
 
 Verify the rsyslog and logrotate rpm configuration is complete by checking the files `etc/rsyslog.d/noobaa_syslog.conf` for rsyslog and `etc/logrotate.d/noobaa/logrotate_noobaa.conf` for logrotate.These files contain the noobaa specific configuration for rsyslog and logrotate.
 
-Rotate the logs manually.
+Logrotate is configured in rsyslog `outchannel`. And script in `logrotate_noobaa.sh` will get executed once `var/log/noobaa.log` size reach 100MB. Both `outchannel` and `size` configured in logrotate `/var/log/noobaa.log` should be same.
+
```

Comment:
Size for outchannel and the size configured in logrotate configuration file
for /var/log/noobaa.log should be same.

Reply:
@aspandey I didnt get you

Reply:
I just wanted to rephrased following line 
"Both `outchannel` and `size` configured in logrotate `/var/log/noobaa.log` should be same."